<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QWidget" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="flow_control" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="b_rewind_b">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>graphics/i_rewind_b.png</normaloff>graphics/i_rewind_b.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_stop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>graphics/i_stop.png</normaloff>graphics/i_stop.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_play">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>graphics/i_play.png</normaloff>graphics/i_play.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_pause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>graphics/i_pause.png</normaloff>graphics/i_pause.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_rewind_f">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>graphics/i_rewind_f.png</normaloff>graphics/i_rewind_f.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="gb_tracker">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Tracker</string>
     </property>
     <widget class="QPushButton" name="b_stop_tracking">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Tracking</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="gb_tracker_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Tracker Type</string>
      </property>
      <widget class="QRadioButton" name="b_goturn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>GOTURN</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_tld">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TLD</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_kcf">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>KCF</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_medianflow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>MEDIANFLOW</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_mil">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>MIL</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_boosting">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>BOOSTING</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="l_info">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
